docker-compose.yml

```diff
@@ -21,6 +21,12 @@ x-app: &app
     - GOOD_JOB_CLEANUP_INTERVAL_SECONDS=86400
     - GOOD_JOB_CLEANUP_PRESERVED_JOBS_BEFORE_SECONDS_AGO=604800
     - HYRAX_ACTIVE_JOB_QUEUE=good_job
+  volumes:
```

Comment:
Will all knapsack projects need these changes? @orangewolf

Reply:
Yes, if they’ve been removed there. Depends on if knapsack got the middle changes or not